src/plugins/discover/public/components/discover_grid/get_render_cell_value.tsx

```diff
@@ -101,14 +101,22 @@ export const getRenderCellValueFn =
           )
         : formatHit(row, dataView, fieldsToShow, maxEntries, fieldFormats);
 
+      const appendWbr = (value: string) => {
+        const wbr = '<wbr/>';
+        return `${value}<wbr>`;
```

Comment:
adding `<wbr>` seems to cause another issue using Safari `15.4`, when switching columns:

https://user-images.githubusercontent.com/463851/159961624-5ea21052-3fd5-4f37-9de4-f15787a12a5d.mp4